wremnants/datasets/datasets2016.py

```diff
@@ -92,8 +143,24 @@ def getDatasets(maxFiles=-1, filt=None):
             xsec = 75.8,
     )
 
-    allPostVFP = [dataPostVFP, WpmunuPostVFP, WmmunuPostVFP, WptaunuPostVFP, WmtaunuPostVFP, ZmmPostVFP, 
-        ZttPostVFP, ttbarlnuPostVFP, ttbarlqPostVFP, wwPostVFP]
+    allPostVFP = [
```

Comment:
I think it would be better to introduce a new list that is only used for the gen, since I think modifying this one could mess up the reco analysis

Reply:
I made a new list for gen sample comparison. RECO studies would not be affected now